src/ts/components/settingspanel.ts

```diff
@@ -91,6 +92,31 @@ export class SettingsPanel extends Container<SettingsPanelConfig> {
     }
   }
 
+  /**
+   * Hack for IE + Firefox
+   * when the settings panel fades out while an item of a select box is still hovered, the select box will not fade out
+   * while the settings panel does. This would leave a floating select box, which is just weird
+   */
+  private hideHoveredSelectBoxes() {
+    this.getItems().forEach((item: SettingsPanelItem) => {
+      if (item.isActive()) {
+        if ((item as any).setting instanceof SelectBox) {
+          const selBox: HTMLElement = (item as any).setting.getDomElement().get()[0];
+          const oldDisplay = selBox.style.display;
+          // updating the display to none marks the select-box as inactive, so it will be hidden with the rest
+          // we just have to make sure to reset this as soon as possible
+          selBox.style.display = 'none';
+          if (window.requestAnimationFrame) {
+            requestAnimationFrame(() => { selBox.style.display = oldDisplay; });
+          } else {
+            // IE9 has no requestAnimationFrame, just use setTimeout(0)
+            setTimeout(() => { selBox.style.display = oldDisplay; }, 0);
```

Comment:
Actually we don't need the setTimeout here as we know that IE does not have the animation frame optimization, so just reset the value directly.